framework/src/main/java/org/checkerframework/framework/stub/RemoveAnnotationsForInference.java

```diff
@@ -350,6 +414,17 @@ static boolean isTrustedAnnotation(String name) {
         || name.equals("org.checkerframework.common.aliasing.qual.LeakedToResult");
   }
 
+  /**
+   * Returns true iff the annotation is present in the user-supplied file of annotations to keep
+   * (via the {@code -keepFile} command-line option).
+   *
+   * @param name the annotation's name (simple or fully-qualified)
+   * @return true if the user requested that this annotation be kept in the source code
+   */
+  static boolean shouldBeKept(String name) {
+    return (annotationsToKeep != null && annotationsToKeep.contains(name));
```

Comment:
Minor: if calling `contains` often on a collection, a `Set` is more appropriate than a `List`.